javascript/javascript2/week3/weatherApp/index.js

```diff
@@ -0,0 +1,150 @@
+//Create your own json file
+
+[
+  {
+      artist: 'Bob Marley',
+      albums: ['burnin', 'catch a fire', 'soul revolution', 'natty dread'],
+      hitsongs: [
+          { title: 'buffalo soldier', length: '6:15', year: 1975 },
+          { title: 'no woman no cry', length: '5:47', year: 1989 },
+      ],
+      genre: 'reggae',
+  },
+  {
+      artist: 'coldplay',
+      albums: ['parachuets', 'xny', 'ghost stories'],
+      hitsongs: [
+          { title: 'yellow', length: '5:15', year: 2000 },
+          { title: 'viva la vida', length: '5:40', year: 2005 },
+      ],
+      genre: 'rock',
+  },
+];
+
+// Find a cool api
+//aniapi.com/docs/
+https: [
+  {
+      status_code: 200, // number
+      message: 'Anime found', // string
+      data: {
+          // object
+          anilist_id: 1,
+          mal_id: 1,
+          format: 0,
+          'status:': 0,
+          titles: {
+              //object inside object
+              en: 'Cowboy Bebop',
+              jp: 'カウボーイビバップ',
+          },
+          // ...
+          id: 0,
+      },
+      version: '1', // The response API version
+  },
+];
+
+//Weather app //be9e1928280950bedf8ff42caa2aa649
+
+const cityInput = document.querySelector('.search-bar');
+const btnToSearchByCity = document.querySelector('.btnSearch');
+
+const cityName = document.querySelector('.cityName');
+const cityTemp = document.querySelector('.temp');
+const feelsLike = document.querySelector('.feels-like');
+const cityIcon = document.querySelector('.icon');
+const windSpeed = document.querySelector('.windSpeed');
+const cloudy = document.querySelector('.cloudy');
+const sunrise = document.querySelector('.sunrise');
+const sunset = document.querySelector('.sunset');
+const description= document.querySelector('.description');
+
+const errorDisplay=document.querySelector('#error-display')
```

Comment:
All of these variables use the same `document.querySelector("something")` right? Why not use a function that would return the value? Would make the code nicer and reusable.